api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -153,9 +154,6 @@ public void shouldUpgradeFromClean1_9To1_10() throws IOException, SQLException {
 		
 		List<Map<String, String>> drugOrderSelect = upgradeTestUtil.select("drug_order", null, "order_id");
 		Assert.assertThat(drugOrderSelect.size(), Matchers.is(0));
-		
-		//Test if the generated schema corresponds to Hibernate mappings
-		upgradeTestUtil.buildSessionFactory();
```

Comment:
Could you remind me as to why we removed the above line?

Reply:
Sure, this is where the validation of the Hibernate mappings happened. They moved to a new test class, namely `ValidateHibernateMappingsDatabaseIT`.